Jenkinsfile

```
@@ -0,0 +1,41 @@
+elifePipeline {
+    def commit
+    def commitShort
+    def schemaVersion
+    def tag
+    DockerImage image
+
+    node('containers-jenkins-plugin') {
+        stage 'Checkout', {
+            checkout scm
+            commit = elifeGitRevision()
+            commitShort = sh(script: "git rev-parse --short HEAD", returnStdout: true).trim()
+            schemaVersion = params.'SCHEMA_VERSION' ?: "master"
```

```suggestion
            schemaVersion = params.SCHEMA_VERSION ?: "master"
```
Quotes should not be needed?

---

(also note the single quotes if interpolation is not needed)